dashboard/package.json

```diff
@@ -45,6 +45,7 @@
     "@angular/platform-server": "^4.3.2",
     "@angular/router": "^4.3.2",
     "@enplug/dashboard-sdk": "^4.1.2",
+    "@enplug/sdk-player": "^0.6.0",
```

Comment:
Why import sdk-player to dashboard app seed?

Reply:
removed